### ISSUE TITLE
Auto Junk Unusable by class

### ISSUE THREAD

**Issue by xenoterracide** [typicalzergling/vendor#75]
So it'd be nice to have a filter to automatically sell things that aren't usable by your class. For example a mage can't use "mail armor" but a mage can use a dagger if you get them trained. On the latter it might be cool if you were able to add a check for the training being present.

I believe [Dejunk](https://www.curseforge.com/wow/addons/dejunk) supports this.

**Comment by fubaWoW** [comment]
You can easy make rules for that!

This Rule will Sell all "Item-Tokens" that are not for your Class:
```LUA
(not IsEquipment) and (Level < 300) and IsSoulbound and (SubType == "Junk")  and TooltipContains("Classes:") and (not TooltipContains(PlayerClass()))
```
maybe need to be adjusted on non English clients and maybe also ItemLevel!

This simple Rule will sell equip that is not for your Class but keeps all Legendaries:
```LUA
IsEquipment and IsSoulbound and (not IsEquippable) and (Quality < 5)
```

**Comment by xenoterracide** [comment]
does TooltipContains work on content added to the tooltip by other addons?

**Comment by fubaWoW** [comment]
> does TooltipContains work on content added to the tooltip by other addons?

I don't think so, because there is not "the only way" to modify and also not "the only way" to read a ToolTip.
It **may** work for AddOn X but not for AddOn Y.
